README.md

```diff
@@ -4,50 +4,74 @@
 CLI tool to scan your postgres database, and output DBML.
 - [pg-to-dbml](#pg-to-dbml)
     - [Description](#description)
-  - [Overview](#overview)
-    - [Resources](#resources)
-    - [Download](#download)
+    - [Tech Stack](#tech-stack)
+    - [Download and install](#download-and-install)
     - [How to use the cli](#how-to-use-the-cli)
     - [Example Usage](#example-usage)
   - [Linting](#linting)
+  - [Debugging](#debugging)
   - [Changelog](#changelog)
 
-## Overview
-
-The purpose of the reference application is to provide a starting point for Node.js development that encapsulates good tools and best practices such as TypeScript, configuration management, linting, testing, and documentation.
-
-### Resources
+### Tech Stack
 
 - [NodeJS](https://nodejs.org/en/docs/)
 - [Yargs](http://yargs.js.org/)
 - [PG](https://node-postgres.com/)
 
-### Download
-`npm i 
+### Download and install
+
+`npm imstall pg-to-dbml` 
```

Comment:
Maybe should be:
```
npm config set @polarislabs:registry https://npm.pkg.github.com/polarislabs
npm install -g @polarislabs/pg-to-dbml
```

Reply:
That shouldn't be necessary since it is a publicly available package on https://npmjs.org.

Reply:
Ah very true, I forgot about that. :+1: